### PR TITLE
[MTM-50749] Updated screenshots for Web SDK - Getting started

### PR DESCRIPTION
**Pull request by pawel-rynarzewski-sag**

### CODE REVIEW

File: content/web/gettingstarted.md [hunk 1]
```diff
@@ -13,11 +13,16 @@ $ npm install -g @c8y/cli
...
 
 ```
-$ c8ycli new myApp
+$ c8ycli new myapp
 ```
 
+It asks a few questions:
+
+- base version: select the one corresponding to the backend version on the target environment
+- base project: select `application`
```
Comment (rssag) [inline content/web/gettingstarted.md:22]:
```suggestion
- base version: select the one corresponding to the backend version on the target environment.
- base project: select `application`.
```

**Review by rssag — APPROVED** [review]

File: content/web/gettingstarted.md [hunk 2]
```diff
@@ -33,7 +38,7 @@ After all packages are installed you can start the application by running:
 $ npm start
 ```
 
-If you point your browser to `http://localhost:9000/apps/myapp/` you will get a login screen which proxies to the tenant defined in the start script. If you cannot log in, it might be pointing to the wrong instance. To change the proxy to your tenant URL change the `start` script in the script section of the newly created package.json:
+If you point your browser to `http://localhost:9000/apps/myapp/` you will get a login screen which proxies to the tenant defined in the start script. If you cannot log in, it might be pointing to the wrong instance. To change the proxy to your tenant URL change the `start` script in the script section of the newly created `package.json`:
```
Comment (BeateRixen) [inline content/web/gettingstarted.md:41]: File and path names are always formatted in italics, so it should be *http://localhost:9000/apps/myapp/* and *package.json*

File: content/web/gettingstarted.md [hunk 3]
```diff
@@ -13,11 +13,16 @@ $ npm install -g @c8y/cli
...
 ```
 
+It asks a few questions:
+
+- base version: select the one corresponding to the backend version on the target environment.
+- base project: select `application`.
+
 {{< c8y-admon-info >}}
-When you want to see the possibilities and implementation details of the Web SDK you should try the tutorial application. You can install it by running `c8ycli new <<your-app-name>> tutorial`.
+If you want to see the possibilities and implementation details of the Web SDK you should try out the tutorial application. You can install it by running `c8ycli new <<your-app-name>> tutorial` or selecting `tutorial` as the base project in the wizard.
```
Comment (BeateRixen) [inline content/web/gettingstarted.md:25]: Sample values which can be selected or entered are always formatted with quotes, so it should be "tutorial" if it refers to a selection in a wizard.

**Review by Airdroplet — APPROVED** [review]

File: content/web/gettingstarted.md (hunk 2 above)
Comment (pawel-rynarzewski-sag) [reply content/web/gettingstarted.md]: ok, applied in all related PRs too (https://github.com/SoftwareAG/c8y-docs/pulls?q=is%3Apr+is%3Aopen+MTM-50749)

File: content/web/gettingstarted.md (hunk 3 above)
Comment (pawel-rynarzewski-sag) [reply content/web/gettingstarted.md]: ok, applied in all related PRs too (https://github.com/SoftwareAG/c8y-docs/pulls?q=is%3Apr+is%3Aopen+MTM-50749)

**Comment by darluk** [comment]
I think that screenshot size can be minimized for https://github.com/SoftwareAG/c8y-docs/pull/858/files#diff-5224b0371f5278b8b768c9b9dc572bde8032cc937fd7873038fbd34665c3cba0, because it seems  like a n empty huge window

**Comment by pawel-rynarzewski-sag** [comment]
@darluk, "screenshot size can be minimized" - done.

**Comment by pawel-rynarzewski-sag** [comment]
@BeateRixen, applied the requested changes, could you please check again and approve?

**Pull request merged**